CONTRIBUTING.md

```diff
@@ -68,60 +73,73 @@ For a complete look at this repository's labels, see the
-new ideas and potential improvements. GitHub Discussions is the best place to post these types of requests
-because it allows for feedback from the entire community and does not bloat the issue tracker. Please note that not
-all feature requests will be incorporated into the code base. Also, feature requests posted on the issue tracker
-will be tagged as `Type: Invalid` and closed. Lastly, please note that spamming the maintainers to incorporate a
-feature you want into any of the charms will not improve its likelihood of being implemented; it may result in you receiving
-a temporary ban from the repository.
+Some common questions you should answer in your report include:
+
+* What is your current environment?
+* Were you able to reproduce the issue in another environment?
+* Which commands/actions/configuration options/etc produce the issue?
+* What was your expected outcome?
+
+Please try to be as detailed as possible in your report. All these details will help the
+maintainers quickly fix issues with the Slurm charms.
+
+## Enhancement Proposals
+
+The Charmed HPC core developers may already know what they want to add to the Slurm charms,
+but they are always open to new ideas and potential improvements. GitHub Discussions is
+a good place for discussing open-ended questions that pertain to the entire Charmed HPC
+project, but more focused enhancement proposal discussion can start within the issue
+tracker.
+
+Please note that not all proposals may be incorporated into the documentation. Also, please
+know that spamming the maintainers to incorporate something you want into the documentation
+will not improve the likelihood of being implemented; it may result in you receiving a
+temporary ban from the repository.
 
 ## Pull Requests
 
-Good pull requests &mdash; patches, improvements, new features &mdash; are a huge help. These pull requests should
-remain focused in scope and should not contain unrelated commits.
+Good pull requests &mdash; patches, improvements, new features &mdash;
+are a huge help. Pull requests should remain focused and not contain commits not
+related to what you are contributing.
 
-__Ask first__ before embarking on any __significant__ pull request (e.g. implementing new features, refactoring code,
-incorporating a new test environment provider, etc.), otherwise you risk spending a lot of time working on something
-that the maintainers might not want to merge into the project! For trivial things, or things that do not require
-a lot of your time, you can go ahead and make a pull request.
+__Ask first__ before embarking on any __significant__ pull request such as add new sections,
+changing the layout, or heavily revising existing sections; otherwise, you risk spending a
+lot of time working on something that Charmed HPC's maintainers may not want to merge into the
+documentation! For trivial changes, or contributions that do not require a large amount of time,
```

Comment:
I'm seeing that this leans heavily on documentation changes, which is partly my fault because I just copied the file verbatim from the docs repo, but can we rewrite it to have more "charmy" development tasks?

Reply:
I'll revise it. Good, can probably lift from a similar CONTRIBUTING.md we have lying around :sweat_smile:

Reply:
Revised this section.